commands/netlify.go

```diff
@@ -33,12 +34,19 @@ var (
 func Execute() {
 	rootCmd.SilenceUsage = true
 
-	rootCmd.PersistentFlags().StringVarP(&endpoint, "endpoint", "E", "https://api.netlify.com", "default API endpoint")
-	rootCmd.PersistentFlags().StringVarP(&auth.AccessToken, "access-token", "A", "", "access token for Netlify's API")
-	rootCmd.PersistentFlags().BoolVarP(&debug, "debug", "D", false, "enable debug tracing")
+	rootCmd.PersistentFlags().StringP("endpoint", "E", "https://api.netlify.com", "default API endpoint")
```

Comment:
It would be better to set the `endpoint` and `streaming` options in a different way so they don't show up as part of the normal CLI options in the help output.

Normal CLI users outside of Netlify should never have any reason to mess with these.